assets/cluster-monitoring-operator/service-monitor.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: cluster-monitoring-operator
```

Comment:
I don't think this `ServiceMonitor` is actually being picked up: https://github.com/openshift/cluster-monitoring-operator/blob/e111e7b3ce84c40f4e336a53d5af3248c2f4ae03/assets/prometheus-k8s/prometheus.yaml#L71-L74

Reply:
hmm .. not sure we can do that, multiple expressions in that array are treated as a logical `AND`.

Reply:
updated the service monitor to use the `k8s-app` label. we will do a follow up to update all labels to use the new openshift keys.